### DISCUSSION TITLE
implementations of features like "Call hierarchy", "find all implementations"

### DISCUSSION THREAD

**Discussion by noone1235** [OmniSharp/omnisharp-vscode#7245] [Q&A] [answered]
I am trying to implement a plugin which can convert C# code to Sequence UML diagram. I want to know if there are any APIs which can help me extract the flow/ sequences of the code easier. 
Something like "find all implementations", "call hierarchy" kind of features that are implemented can help me a lot, are these features implemented in this extension?

**Comment by Ador-able** [comment]
Hope to get support for Call hierarchy.

**Comment by dibarbet** [comment, marked as answer]
Call hierarchy is tracked here - https://github.com/dotnet/vscode-csharp/issues/6297 (if you'd like this, please upvote).

Find all implementations should already be supported.
